flink-core/src/test/java/org/apache/flink/api/common/typeutils/SerializerTestBase.java

```diff
@@ -99,6 +104,7 @@ public void testCopy() {
 			
 			for (T datum : testData) {
 				T copy = serializer.copy(datum);
+				String str = copy.toString();
```

Comment:
This causes a lot of warnings for me. Can we change that to simply `copy.toString()` without declaring an unused variable to hold the result?